content/manuals/engine/storage/drivers/windowsfilter-driver.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Learn about the windowsfilter storage driver
+keywords: container, storage, driver, windows, windowsfilter
+title: windowsfilter storage driver
+---
+
+The windowsfilter storage driver is the default storage driver for Docker
+Engine on Windows. The windowsfilter driver uses Windows-native file system
+layers to for storing Docker layers and volume data on disk. The windowsfilter
+storage driver only works on file systems formatted with NTFS.
```

Comment:
I don't know if there's more in-depth information about the underlying technology (perhaps @StefanScherer knows?).

We could provide a link to Microsoft's container documentation, which has some (but limited) info; https://learn.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/container-storage

Reply:
Added a link to that page.

Reply:
I don't have more information, but thanks, the link is a good idea.